src/core/streams/stream_actions.ts

```diff
@@ -1,10 +1,18 @@
 import { StreamElement } from "../../elements/stream_element"
 
 export const StreamActions: { [action: string]: (this: StreamElement) => void } = {
+  after() {
+    this.targetElement?.parentElement?.insertBefore(this.templateContent,this.targetElement.nextSibling)
```

Comment:
I think `nextSibling` could be `undefined` when there is only one element present. I suggest using the `insertAdjacentHTML` API for both actions. https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML

Reply:
Hi,
It can be `null` with insertBefore, this is well handle.
[https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore](https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore)

`If this is null, then newNode is inserted at the end of parentNode's child nodes.`

insertAdjacentHTML seems not to be a good choice. Templates content are `DocumentFragment`, so insertAdjacentElement could be a better choice. But it's less compliant with all browser.

Reply:
Cool. I didn't know about that `null` detail.